lib/plugins/aws/utils/format-lambda-log-event.js

```diff
@@ -16,13 +16,15 @@ module.exports = (msgParam) => {
 
   if (msg.startsWith('REPORT')) {
     const splitted = msg.split('\t');
-    const durationItem = splitted[1];
-    const maxMemoryUsedItem = splitted[4];
-    const initDurationItem = splitted[5];
+    const duration = splitted[1];
+    const maxMemoryUsed = splitted[4].slice(4);
+    const initDuration = splitted[5] && splitted[5].split(':')[1];
     // Simplify the output and trim out unnecessary information
-    msg = `END ${durationItem} (init:${initDurationItem.split(':')[1]}) ${maxMemoryUsedItem.slice(
-      4
-    )}`;
+    if (initDuration) {
+      msg = `END ${duration} (init:${initDuration}) ${maxMemoryUsed}`;
```

Comment:
```suggestion
      msg = `END ${duration} (init: ${initDuration}) ${maxMemoryUsed}`;
```

should we fix the spacing in this PR?

Reply:
it is correct, the split that produces `initDuration` already has a leading space

Reply:
ah thanks, got it!